src/main/scripts/org/reaktivity/specification/nukleus/http/streams/rfc7230/flow.control/response.headers.too.long/server/target.rpt

```diff
@@ -0,0 +1,66 @@
+#
+# Copyright 2016-2017 The Reaktivity Project
+#
+# The Reaktivity Project licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+property nuklei ${nuklei:directory("target/nukleus-itests").streamsCapacity(1024 * 1024, 64 * 1024)}
+
+property targetInput ${nuklei.streams("target", "http#source")}
+property targetOutputEst ${nuklei.streams("http", "target")}
+
+property newTargetOutputEstId ${nuklei:newStreamId()}
+
+connect await ROUTED_INPUT
+        agrona://stream/bidirectional
+        option reader ${agrona:oneToOneReader(targetInput.buffer)}
+        option writer ${agrona:oneToOneWriter(targetInput.throttle)}
+connected
+
+read [0x00 0x00 0x00 0x01]
+read ([0..8]:targetInputId)
+read ([0..8]:targetInputRef)
+read ([0..8]:targetCorrelationId)
+read [0x00] [0x07] ":scheme" [0x04] "http"
+read [0x00] [0x07] ":method" [0x03] "GET"
+read [0x00] [0x05] ":path" [0x01] "/"
+read [0x00] [0x0a] ":authority" [0x0e] "localhost:8080"
+
+read notify HTTP_REQUEST_RECEIVED
+
+read [0x00 0x00 0x00 0x03]
+read ${targetInputId}
+
+close
+closed
+
+connect await HTTP_REQUEST_RECEIVED
+        agrona://stream/bidirectional
+        option reader ${agrona:oneToOneReader(targetOutputEst.throttle)}
+        option writer ${agrona:oneToOneWriter(targetOutputEst.buffer)}
+connected
+
+write [0x00 0x00 0x00 0x01]
+write ${newTargetOutputEstId}
+write [0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00]
+write ${targetCorrelationId}
+write [0x00] [0x07] ":status" [0x03] "200"
+write [0x00] [0x10] ":Custom-Header-1" [0x17] "customer header 1 value"
+write [0x00] [0x10] ":Custom-Header-2" [0x17] "customer header 2 value"
```

Comment:
_customer_ header typo in name
The leading colon should not be present for custom headers as that is for pseudo-headers like `:status` only.